#!/bin/sh /usr/share/dpatch/dpatch-run
## 201-magic-update-avi.dpatch by root <ap@insysnet.ru>
##
## DP: Adds magic for H.264 video in AVI files (Closes: #347263).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/riff file-4.20/magic/Magdir/riff
--- file-4.20.orig/magic/Magdir/riff	2006-11-17 16:09:54.000000000 +0000
+++ file-4.20/magic/Magdir/riff	2007-04-03 15:49:16.000000000 +0000
@@ -130,6 +130,7 @@
 >>>>>>>(104.l+132)      string/c        divx    DivX 4
 >>>>>>>(104.l+132)      string/c        dx50    DivX 5
 >>>>>>>(104.l+132)      string/c        xvid    XviD
+>>>>>>>(104.l+132)	string/c	h264	H.264
 >>>>>>>(104.l+132)      string/c        wmv3    Windows Media Video 9
 >>>>>>>(104.l+132)      string/c        h264    X.264
 >>>>>>>(104.l+132)      lelong  0
